deepchem/utils/test/test_pytorch_utils.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+import pytest
+try:
+    import torch
+    has_torch = True
+except ModuleNotFoundError:
+    has_torch = False
+
+from deepchem.utils.pytorch_utils import unsorted_segment_sum
+
+
+@pytest.mark.torch
+def test_unsorted_segment_sum():
```

Comment:
Can you add a test about the claim about segment_ids.shape is a prefix of data.shape?